### PR TITLE
feat: Add storybook configs

### PR DESCRIPTION
**Pull request by ArthurTriis1**
Signed-off-by: Arthur Andrade <arthurfelandrade@gmail.com>

## How does it work?

This PR adds Storybook in the base.store. The Storybook was added because the devs needed an environment to tests their components don't needing to add the component in a page.

Screen Shot 2022-04-06 at 16 29 04

## How to test it?

Run in terminal `yarn storybook` and, after the build, open `localhost:6006`

## References

[TsconfigPathsPlugin](https://stackoverflow.com/a/70272640)
[Installing graphql-utils on gatsby](https://github.com/vtex/faststore/tree/master/packages/graphql-utils#installing-on-gatsby)
[Theming Storybook](https://storybook.js.org/docs/react/configure/theming)
[Gatsby with Storybook](https://www.gatsbyjs.com/docs/how-to/testing/visual-testing-with-storybook/)

## Checklist

<em>You may erase this after checking them all ;)</em>

- [x] CHANGELOG entry added

### CODE REVIEW

File: src/components/ui/Badge/Badge.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <Badge {...args}>{children}</Badge>
+)
+
+export const Default = Template.bind({})
+
+Default.args = {
+  children: 'Something',
+  small: false,
+  interactive: false,
+  variant: 'highlighted',
```
Comment (filipewl) [inline src/components/ui/Badge/Badge.stories.tsx:20]: So the example looks like the discount badge we have on the PDP:
```suggestion
  children: '50% off',
  small: true,
  interactive: false,
  variant: 'neutral	',
```

File: src/components/ui/Breadcrumb/Breadcrumb.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    position: 2,
+  },
+  {
+    item: 'item3',
+    name: 'item3 There are many variations of passages of Lorem',
+    position: 3,
+  },
+  {
+    item: 'item4',
+    name: 'item4 There are many variations of passages of Lorem',
```
Comment (filipewl) [inline src/components/ui/Breadcrumb/Breadcrumb.stories.tsx:23]:
```suggestion
  { item: 'technlogy', name: 'Technology', position: 1 },
  {
    item: 'technology/electronics',
    name: 'Electronics',
    position: 2,
  },
  {
    item: 'technology/electronics/audio-and-video',
    name: 'Audio & Video',
    position: 3,
  },
  {
    item: 'technology/electronics/audio-and-video/headphones',
    name: 'Headphones',
```

File: vtex.env [hunk 3]
```diff
@@ -4,7 +4,7 @@ ENABLE_GATSBY_REFRESH_ENDPOINT=true
 GATSBY_QUERY_ON_DEMAND_LOADING_INDICATOR=false
 
 # configures WebOps
-USE_NODE_MODULES_CACHE=true
+USE_NODE_MODULES_CACHE=false
```
Comment (filipewl) [inline vtex.env:7]:
```suggestion
# Set to false while the WebOps team fixes the Jira's DXWP-279 issue.
USE_NODE_MODULES_CACHE=false
```

**Review by filipewl — APPROVED** [review]
_Habemus Storybook!_ 
🙌 

Left some suggestions to update the examples with a more realistic content/copy.

File: src/components/ui/Breadcrumb/Breadcrumb.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+)
+
+export const WithDropdown = Template.bind({})
+
+WithDropdown.args = {
+  breadcrumbList: [
+    ...breadcrumbList,
+    {
+      item: 'item5',
+      name: 'item5 There are many variations of passages of Lorem',
```
Comment (filipewl) [inline src/components/ui/Breadcrumb/Breadcrumb.stories.tsx:39]:
```suggestion
      item: 'technology/electronics/audio-and-video/headphonesaedle-vk1-headphone',
      name: 'Aedle VK-1 L Headphone',
```

**Review by israelswf — APPROVED** [review]
Amazing PR, Arthur ✨

**Pull request merged**